confgenerator/confgenerator_test.go

```diff
@@ -148,11 +175,11 @@ func readFileContent(testName string, filePathFormat string, t *testing.T, respe
 	return rawExpectedConfig
 }
 
-func updateOrCompareGolden(t *testing.T, testName string, expectedBytes []byte, actual string, path string) {
+func updateOrCompareGolden(t *testing.T, testName, goos string, expectedBytes []byte, actual string, path string) {
```

Comment:
The convention here seems to be to specify the type for each parameter, even if they repeat. Can we please make this consistent?